resources/META-INF/plugin.xml

```diff
@@ -12,7 +12,7 @@
     ]]></description>
 
   <change-notes><![CDATA[
-        <li><b>0.3.2-beta1</b>: Introduced class members in completion list</li>
+        <li><b>0.3.2-alpha</b>: Introduced class members in completion list, improved syntax support</li>
```

Comment:
Actually there is `0.3.2-beta1`: https://github.com/phalcon/zephir-idea-plugin/releases/tag/0.3.2-beta1

I propose to release next beta version, `0.3.2-beta2`